healthcheck/api/bnd.bnd

```diff
@@ -1,12 +1,6 @@
-Bundle-Category: healthcheck
-
-Bundle-Description: ${project.description}
-
-Bundle-DocURL: https://felix.apache.org
-
-Bundle-License: Apache License, Version 2.0
-
-Bundle-Vendor: The Apache Software Foundation
-
-Export-Package: org.apache.felix.hc.api.*
-
+Bundle-Category     : healthcheck
+Bundle-Description  : ${project.description}
+Bundle-DocURL       : https://felix.apache.org
+Bundle-License      : Apache License, Version 2.0
+Bundle-Vendor       : The Apache Software Foundation
+Export-Package      : org.apache.felix.hc.api.*
```

Comment:
No change required in api (commons-lang is not used)

Reply:
I just wanted to make the bnd files look alike. There is no real change in the API bundle.

Reply:
Best to keep formatting changes out of PRs (because they are often based on personal preferences)